src/components/media/MediaMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+import IconEllipsis from '../../icons/general/IconEllipsis';
+import { ButtonType } from '../button';
+import PlainButton from '../plain-button';
+import { PlainButtonProps } from '../plain-button/PlainButton';
```

Comment:
Q: Wasn't sure it's better to expose this in `plain-button/index.ts` or not.

Reply:
Yeah, I think it would be good to export the props in `plain-button/index.ts`